charts/oauth2-proxy-manifests/templates/deployment.yaml

```diff
@@ -68,9 +72,17 @@ spec:
           mountPath: /etc/config/script/iat_to_clienid.sh
           subPath: iat_to_clienid.sh            
       containers:
-{{- if .Values.enableLogging }}
+{{- if (((.Values.global.cp).logging).fluentbit).enabled }}
```

Comment:
here and later: if parenthesizing is really needed then either `global.cp` is non-standard or there must be something wrong with the language...